mobile/lib/features/brag/brag.dart

```diff
@@ -0,0 +1,426 @@
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.only(left: 8),
+                            child: Row(
+                              children: [
+                                Text(
+                                  pnl.formatted(),
+                                  style: primaryTextStypeValue,
+                                ),
+                                Icon(
+                                  BitcoinIcons.satoshi_v2,
+                                  color: pnlColor,
+                                  size: 20,
+                                ),
+                              ],
+                            ),
+                          ),
+                          Padding(
+                            padding: const EdgeInsets.only(left: 8),
+                            child: Row(
+                              children: [
+                                Text(
+                                  pnlPercent.toString(),
+                                  style: primaryTextStypeValue,
+                                ),
+                                Icon(
+                                  Icons.percent,
+                                  color: pnlColor,
+                                  size: 20,
+                                ),
+                              ],
+                            ),
+                          )
+                        ],
+                      ),
+                    ),
+                    const SizedBox(
+                      height: 10,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(left: 8, bottom: 20),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Row(
+                            children: [
+                              const Text(
+                                "Side",
+                                style: secondaryTextHeading,
+                              ),
+                              const SizedBox(
+                                width: 5,
+                              ),
+                              Text(
+                                "${leverage.formattedReverse()} ${direction.nameU}",
+                                style: secondaryTextValue,
+                              )
+                            ],
+                          ),
+                          Row(
+                            children: [
+                              Padding(
+                                padding: const EdgeInsets.only(left: 8),
+                                child: Row(
+                                  children: [
+                                    const Text(
+                                      "Entry price",
```

Comment:
🙃 The entry price is kind of interesting, but I might just omit it too in favour of keeping the image less cluttered.

I think the story the user might want to tell is something as simple as "I went long on Bitcoin at leverage 50 and I made this much".

Reply:
Fair, I'll remove it